dlk/hw/make_os/build/make_rootfs.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+if [ $# -ne 1 ]; then
+	echo "Error: Number of argument should be 1"
+	exit 0
+fi
+
+BIT_32_OR_64=$1
+
+if [ $1 = 32 ]; then
+	DEBOOTSTRAP_ARCH=armhf
+	QEMU_ARCH=arm
+	OUTPUT_FNAME=rootfs32.tgz
+elif [ $1 = 64 ]; then
+	DEBOOTSTRAP_ARCH=arm64
+	QEMU_ARCH=aarch64
+	OUTPUT_FNAME=rootfs64.tgz
```

Comment:
In my opinion, considering the case when we need add other architecture, it is better to specify the architecture `armhf`, `arm64` directly instead of `32` or `64`, and output files is better to named as  `rootfs_armhf.tgz` and `rootfs_arm64.tgz` 
What do you think?